inst/templates/core/01_prepr.qmd

```diff
@@ -0,0 +1,154 @@
+---
+title: "01_prepr"
+format: html
+editor: visual
+---
+
+## 1. PEM Preparation Steps
+
+This template provides a guide to help guide you through the PEM workflow.
+
+## Read in library 
+
+When you click the **Render** button a document will be generated that includes both content and the output of embedded code. You can embed code like this:
+
+```{r,eval=FALSE}
+library(PEMprepr)
+
+project_name <- "{{ aoi_name }}"
+
+fid <- read_fid()
+
+```
+
+## Prepare base data
+
+Set up base spatial file to define the area of interest.
+
+If you have previously supplied a spatial file when creating the PEMr project, you will find a copy of your spatial file located within the folder "00_raw_inputs/10_vector/". 
```

Comment:
```suggestion
```

As I mentioned in #4, I think this is the kind of content that would be in the vignette, but not necessarily in the templates - it gives package usage instructions rather than analysis narrative...

Reply:
Yes I agree, I think I will push this to the .Rmd